app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
```

Comment:
Start every argument from a new line. Check other `__init__` method for this mistake